tests/ROCrate/Main.fs

```diff
@@ -3,18 +3,19 @@ module ARCtrl.ROCrate.Tests
 open Fable.Pyxpecto
 
 let all = testSequenced <| testList "ROCrate" [
-    Tests.LDObject.main
-    Tests.Dataset.main
-    Tests.Investigation.main
-    Tests.Study.main
-    Tests.Assay.main
-    Tests.LabProcess.main
-    Tests.LabProtocol.main
-    Tests.Sample.main
-    Tests.Data.main
-    Tests.PropertyValue.main
-    Tests.Person.main
-    Tests.ScholarlyArticle.main
+    Tests.LDContext.main
+    Tests.LDNode.main
+    //Tests.Dataset.main
```

Comment:
hmm i think i found the issue, you are not running the tests. Why fix them then?